### PR TITLE
Add optional features as list of input boxes

### PR DESCRIPTION
**Pull request by XHatan**
**Issue #:**
additional fix for 

https://github.com/aws/amazon-chime-sdk-js/pull/815 

**Description of changes:**

Change selection box to list of input boxes to allow selection of combination of optional features.
Also change the simulcast integration test based on the change.

**Testing**

1. Have you successfully run `npm run build:release` locally? yes
2. How did you test these changes? 

1). manual testing with demo page
2) local grid and KITE auto testing - box is checked and log show simulcast is selected.
 - ```$KITE_HOME/scripts/mac/path/r configs/video_test_simulcast.config.json```

3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? yes
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? no
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? no

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: integration/js/pages/AppPage.js [hunk 1]
```diff
@@ -77,9 +81,24 @@ class AppPage {
     await authenticateButton.click();
   }
 
+  async chooseUseWebAudio() {
+    let webAudioFeature = await this.driver.findElement(elements.webAudioFeature);
+    let webAudioFeatureLabel = await this.driver.findElement(elements.webAudioFeatureLabel);
+    if (await webAudioFeature.isSelected()) {
+         console.log('Web Audio is selected');
```
Comment (richnew10) [inline integration/js/pages/AppPage.js:88]: Nit: indenting. I can fix this when I come by here again.

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 2]
```diff
@@ -318,28 +326,6 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
...
-      this.enableWebAudio = false;
-      this.enableUnifiedPlanForChromiumBasedBrowsers = true;
-      this.log("Feature lists:");
-      for (let i = 0; i < collections.length; i++) {
-        // hard code magic
-        if (collections[i].value === 'simulcast') {
-          this.enableSimulcast = true;
-          this.log('attempt to enable simulcast');
-          const videoInputQuality = document.getElementById('video-input-quality') as HTMLSelectElement;
-          videoInputQuality.value = '720p';
```
Comment (richnew10) [inline demos/browser/app/meetingV2/meetingV2.ts:334]: Do we no longer need to do this?

File: integration/js/pages/AppPage.js (hunk 1 above)
Comment (XHatan) [reply integration/js/pages/AppPage.js]: updated

File: demos/browser/app/meetingV2/meetingV2.ts (hunk 2 above)
Comment (XHatan) [reply demos/browser/app/meetingV2/meetingV2.ts:334]: updated

**Review by richnew10 — APPROVED** [review]
So this is fine, but that nothing broke implies that there's no integration test coverage for simulcast itself — to test that when we try to enable simulcast we actually get it. Seems like we should address that.

**Comment by XHatan** [comment]
> So this is fine, but that nothing broke implies that there's no integration test coverage for simulcast itself — to test that when we try to enable simulcast we actually get it. Seems like we should address that.

True. The feature testing relies on a button to be clicked. If demo page is changed, we might slip again and not get test coverage.

**Pull request merged**